apps/docs/pages/guides/cli.mdx

```diff
@@ -1,162 +1,47 @@
 import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
-  title: 'Supabase CLI',
-  description:
-    'The Supabase CLI provides tools to develop your project locally and deploy to the Supabase Platform.',
-  subtitle:
-    'The Supabase CLI provides tools to develop your project locally and deploy to the Supabase Platform.',
+  id: 'cli',
+  title: 'Local Dev / CLI',
```

Comment:
Nitpick: the forward slash feels a bit out of place. How about 

```suggestion
  title: 'Local Dev with CLI',
```

Or perhaps just Local Dev?